projects/vdk-core/.gitlab-ci.yml

```diff
@@ -26,8 +26,11 @@ image: "python:3.9"
     changes: *vdk_core_locations
   artifacts:
     when: always
+    paths:
+      - ./projects/vdk-core/tests.xml
+    expire_in: 1 week
     reports:
-      junit: tests.xml
+      junit: ./projects/vdk-core/tests.xml
```

Comment:
this change is not quite described anywhere within the PR. could you please elaborate?

Reply:
I added while troubleshooitng. 

Fixed the path to tests.xml so that it shows properly in the test tab - https://gitlab.com/vmware-analytics/versatile-data-kit/-/pipelines/712924440/test_report 

And wanted to be able to download the XML test report